main.js

```diff
@@ -109,6 +96,28 @@ function displayLocalStorageCards() {
     for (var i = 0; i < fromStorage.length; i++) {
       createIdea(fromStorage[i].id, fromStorage[i].title, fromStorage[i].body, fromStorage[i].starred)
```

Comment:
Passing the `starred` value here to `createIdea` has no effect since you've defaulted your `starred` property in `idea.js` to `false`. Additionally, `createIdea` doesn't have a fourth parameter.

While I still think you should update the changes you made in `idea.js` to accurately update the class, should you decide to keep those changes, you should remove the passing of this argument.

Reply:
Gotcha. I'll remove it and test.